src/api.ts

```diff
@@ -381,7 +401,7 @@ export interface StatusData extends BaseStatusData {
   receiving: PendingReceivingInfo
 }
 
-export interface FailedStatusData {
+export interface FailedStatusData extends BaseStatusData {
   status: 'FAILED'
   sending: BaseTransactionInfo
 }
```

Comment:
I guess here we could also overwrite `substatus: Substatus` with `substatus: SubstatusFailed` to make it more restrictive? But I think we are not doing this for the other states, so not hard requirement

Reply:
Hmm... if so, I'd create the dedicated type for each substatus. But not sure if we want it 😅🤔